packages/@react-native-windows/cli/src/runWindows/runWindows.ts

```diff
@@ -4,6 +4,48 @@
  * @format
  */
 
+import * as path from 'path';
+import * as fs from 'fs';
+import {randomBytes} from 'crypto';
+import * as appInsights from 'applicationinsights';
+
+appInsights.setup('795006ca-cf54-40ee-8bc6-03deb91401c3');
+const telClient = appInsights.defaultClient;
+telClient.commonProperties.sessionId = randomBytes(16).toString('hex');
```

Comment:
I see quite a bit of duplication, should we have a telemetry helper library?

Reply:
I like this idea. That would also make it pretty easy to add an env variable to opt-out of telemetry in addition to the CLI flag.

Reply:
We can follow up offline on this.